app/assets/javascripts/upload-spinner.js

```diff
@@ -14,7 +14,7 @@ $("#uploadForm").submit(function(e){
         };
 
         function addUploadSpinner(){
-            $("#processing").append('<p class="govuk-visually-hidden">'+$("#processingMessage").val()+'</p><div><svg class="ccms-loader" height="100" width="100"><circle cx="50" cy="50" r="40"  fill="none"/></svg></div>')
+            $("#processing").append('<p class="govuk-visually-hidden" aria-live="polite">'+$("#processingMessage").val()+'</p><div><svg class="ccms-loader" height="100" width="100"><circle cx="50" cy="50" r="40"  fill="none"/></svg></div>')
```

Comment:
@BriWak does this need to be applied to the `second-spinner.js` too?

Reply:
@koladeadewuyi It will need to be if it works, I'm just doing it for file upload first so that it can be tested in staging. If it works I'll apply it to the second spinner as well.

Reply:
Okay. That makes sense.